snapshotter/demux/metrics/proxy.go

```diff
@@ -30,10 +32,76 @@ const (
 	MaxMetricsResponseSize = 32768 // 32 KB
 )
 
+// Monitor is used for port and lifecycle management of metrics proxies.
+type Monitor struct {
+	// monitorChan is used for communicating when a metrics proxy and its port are no longer in use
+	// so that the Monitor can mark the port as free.
+	ch chan int
```

Comment:
ch or monitorChan?

Reply:
ah thanks 🤦 `monitor.monitorChan` is a bit verbose, updated godoc to `ch`